### DISCUSSION TITLE
TWiLight Menu++中文使用指南（部分翻译）

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/xzonn.github.io#17] [General]
# TWiLight Menu++中文使用指南（部分翻译）

TWiLight Menu++中文使用指南（部分翻译） - 来自Xzonn的小站

https://xzonn.top/posts/TWiLightMenu-Guide.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****嗯 评论于 2023-09-01 00:43:44 (UTC+8)*****

此方法已失效，最新版的twilight menu没有Widescreen文件夹或者bin文件

**Comment by YFMX167** [comment]
twi系统似乎对rom目录下文件命名有要求，中文会导致识别不出来
